.github/workflows/build.yml

```diff
@@ -25,7 +28,8 @@ jobs:
       run: |
         mkdir client/build
         cd client/build
-        CC=/usr/bin/${{ matrix.cc }} cmake ..
+        CC=/usr/bin/${{ matrix.cc }}
```

Comment:
You can't put this on a separate line unless you prefix it with `export`. The CI build is currently using `/usr/bin/cc` as the C compiler for both GCC and Clang builds.

Reply:
Fixed.